src/bosh_azure_cpi/spec/integration/cert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
The cert is ssh_certificate in azure properties.
I think it can be kept because it is just used in our CI VMs.
Or we can remove it with code, and generate it in CI dynamically.